src/CommonITILObject.php

```diff
@@ -870,14 +882,13 @@ public function isUser($type, $users_id)
     /**
      * Is a group linked to the object ?
      *
-     * @param integer $type      type to search (see constants)
-     * @param integer $groups_id group ID
+     * @param int $type      type to search (see constants)
+     * @param int $groups_id group ID
      *
-     * @return boolean
+     * @return bool
      **/
-    public function isGroup($type, $groups_id)
+    public function isGroup(int $type, int $groups_id): bool
```

Comment:
Not sure this won't cause issues; I think we should avoid this kind of changes in a bugfixes release.

Reply:
Indeed, let's be safe.
Done ->[8d6219a](https://github.com/glpi-project/glpi/pull/12437/commits/8d6219af72b27b54e19a37958c372121c10ef4b3)